### ISSUE TITLE
Parancs cooldown

### ISSUE THREAD

**Issue by GamerTomii** [FightMan01/fightman01bot#205]
**Kérlek írd le az ötlet részleteit.**
Parancs használati cooldown, a parancs spam, és így az esetleges rate limit elkerülése érdekében.

**Comment by 182marci** [comment]
+1

**Comment by GamerTomii** [comment]
Khm

**Comment by InvisibleMan780** [comment]
Szerintem  őegy ilyen ?!botslowmode idő re gondol

**Comment by GamerTomii** [comment]
Nem. Érthetően fogalmaztam meg, a fejlesztők meg is értették, de te nem.
És ez nem egy parancs...

**Comment by TancsicsGergely** [comment]
Igazából már van pár parancs amire be van vezetve, viszont mi inkább egy másik megoldást keresnénk, mert meg kell hagyni, néha idegesítő a cooldown...

**Comment by GamerTomii** [comment]
Sad face
